packages/webapp/public/locales/en/animal.json

```diff
@@ -17,6 +17,22 @@
     "BORN_AT_FARM": "Born at the farm",
     "BROUGHT_IN": "Brought in"
   },
+  "PURPOSE": {
+    "GRAZING": "Grazing",
+    "MILKING": "Milking",
+    "MANURE_MANAGEMENT": "Manure management",
+    "PEST_OR_DISEASE_CONTROL": "Pest or disease control",
+    "BREEDING": "Breeding",
+    "FEEDING": "Feeding",
+    "VETERINARY_CARE": "Veterinary care",
+    "BIRTHING": "Birthing",
+    "WEANING": "Weaning",
+    "TRANSPORTATION": "Transportation",
+    "BEHAVIORAL_AND_SOCIAL_MANAGEMENT": "Behavioral and social management",
```

Comment:
This one is _extremely_ NIT and totally optional, but in Canada, behavioural is spelt with a "u"

Reply:
Ohhh interesting! Does Canada use UK spelling for all words?

Reply:
Unfortunately we don't, we use some American 😆🥲 It depends on the field and I heard it has something to with historical ties, e.g. that we use "tire" and not "tyre" because of automotive industry links or such. But for the most part UK is more common.